### PR TITLE
enhance: add progress bar and spaces between table

### PR DESCRIPTION
**Pull request by Abizrh**

### CODE REVIEW

File: lua/pomodoro/ui.lua [hunk 1]
```diff
@@ -64,65 +55,74 @@ local function createBufferOpts()
         height = win_height,
         row = row,
         col = col,
-        border = "single",
+        border = "rounded",
     }
     return opts
 end
 
----@class PomodoroUI
```
Comment (QuentinGruber) [inline lua/pomodoro/ui.lua:72]: Why removing these comments ?

File: lua/pomodoro/ui.lua [hunk 2]
```diff
@@ -64,65 +55,74 @@ local function createBufferOpts()
...
-    return s
+local function progress_bar(current, total, width)
+    local filled = math.floor(current / total * width)
+    return "["
+        .. string.rep("=", filled)
+        .. string.rep(" ", width - filled)
+        .. "]"
 end
 
----@param pomodoro Pomodoro
```
Comment (QuentinGruber) [inline lua/pomodoro/ui.lua:93]: .

File: lua/pomodoro/ui.lua [hunk 3]
```diff
@@ -64,65 +55,74 @@ local function createBufferOpts()
...
 end
+
 function UI.isWinOpen()
     if UI.win == nil then
         return false
     end
     return vim.api.nvim_win_is_valid(UI.win)
 end
--- update the UI time and in which phase we are
----@param pomodoro Pomodoro
```
Comment (QuentinGruber) [inline lua/pomodoro/ui.lua:125]: .

File: lua/pomodoro/ui.lua [hunk 4]
```diff
@@ -136,13 +136,14 @@ function UI.updateUi(pomodoro)
         end
     end)
 end
----@param pomodoro Pomodoro
```
Comment (QuentinGruber) [inline lua/pomodoro/ui.lua:139]: .

**Review by QuentinGruber — CHANGES_REQUESTED** [review]
I'm wondering why you removed certain lua type comments, if you can shed some light on this? Otherwise it's all good :)

File: lua/pomodoro/ui.lua (hunk 1 above)
Comment (Abizrh) [reply lua/pomodoro/ui.lua:72]: well its basically my first time using lua and play around with nvim plugin :) . so I did not notice about it

**Review by QuentinGruber — APPROVED** [review]

**Comment by QuentinGruber** [comment]
thx @Abizrh :) !

**Comment by Abizrh** [comment]
> thx @Abizrh :) !

sure :+1:

**Pull request merged**